test.sh

```diff
@@ -83,9 +87,17 @@ npm run test
 rc=$?;
 cd ..
 
+# Run coverage
```

Comment:
It's a total nit but we already ran with coverage. It would be nice if this comment explained we are merging the coverage from the unit tests and the REST tests.